### ISSUE TITLE
Switch from mocha-eslint to eslint CLI?

### ISSUE THREAD

**Issue by pdehaan** [mozilla/tofino#701]
Not sure if there is a benefit to using the ESLint stuff via Mocha versus just using it directly, but in my very informal local testing, it looks like it's about 3x faster to just use `$ eslint .` directly instead of linting via Mocha.

```sh
$ time npm test test/unit/lint/test-eslint.js
... # About 184 lines of output streamed to console/Travis-CI logs.
# 21.96s user
#  1.72s system
# 105% cpu
# 22.495 total
```

versus:

```sh
$ time ./node_modules/.bin/eslint .
# 7.42s user
# 0.27s system
# 108% cpu
# 7.065 total
```

**Comment by pdehaan** [comment]
Unrelated, but curious why we do something async+promises like this:

```js
const manifest = await fs.readJson('package.json');
```

versus just importing the JSON via an `import` at the top of the test/unit/lint/*.js files:

```js
import manifest from '../../../package.json';
```

**Comment by Mossop** [comment]
I would review a patch to do this, I've seen mocah-eslint fail with a timeout many times in CI so I want to get rid of it.

**Comment by Mossop** [comment]
This was fixed by 9375a82acd1f2fff2ba5c453c577360d964cd875
